src/hotspot/share/gc/g1/g1MMUTracker.cpp

```diff
@@ -111,46 +111,55 @@ void G1MMUTracker::add_pause(double start, double end) {
   }
 }
 
+//                                          current_timestamp
+//                   GC events             /  pause_time
+//                 /     |     \          | /  /
+// -------------[----]-[---]--[--]--------|[--]-----> Time
+//              |         |                   |
+//              |         |                   |
+//              |<- limit |                   |
+//              |         |<- deficit         |
+//              |         ^                   |
+//              |                             |
+//              |<------- _time_slice   ----->|
+//
+// Focusing on GC events that are inside [limit, current_timestamp], we iterate
+// over them from the newest to the oldest (right-to-left in the diagram) and
+// try to locate the timestamp annotated with ^, so that the accumulated GC
+// time inside [deficit, current_timestamp], is equal to gc_budget. Next,
```

Comment:
`gc_budget` is not defined anywhere in this comment or the diagram. This is confusing.